doc/api/tls.md

```diff
@@ -1406,6 +1406,10 @@ changes:
     order as their private keys in `key`. If the intermediate certificates are
     not provided, the peer will not be able to validate the certificate, and the
     handshake will fail.
+  * `sigalgs` {string}` Colon-separated list of supported signature algorithms.
```

Comment:
I feel like it might be a tad more user-friendly if we supported an array of strings?

Reply:
@addaleax Thank you for the review!

Can be definitely done, but I was trying to use the same style as with somewhat related `ciphers` option.

Reply:
I agree that an array should be allowed, though the `:` seperated form is common, and is used elsewhere in our API. Perhaps allow either?

Reply:
I think consistency is most compelling, and simplicity, so lets leave as-is, `:` seperated. Looks like @addaleax is OK with that, too.

Reply:
Yeah, I think either using `:`-separated strings only or adding support for both variants (and then also doing that for `ciphers`) is fine.

Reply:
ok, agreed on `:` for now, someone else can change these to allow arrays at some point if they want.